src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();
```

Comment:
The 'fruits' map should be private to adhere to encapsulation. Public static fields can be modified from anywhere in the code, which can lead to hard-to-track bugs and breaches in the data integrity.